### ISSUE TITLE
PIP installation of wavetrace doesn't work

### ISSUE THREAD

**Issue by kesara** [NZRS/wavetrace#19]
README file suggests to install wavetrace with pip, but doesn't work.

```
Collecting wavetrace
  Downloading wavetrace-3.0.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-20787n9x/wavetrace/setup.py", line 11, in <module>
        long_description=open('README.rst').read(),
    FileNotFoundError: [Errno 2] No such file or directory: 'README.rst'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-20787n9x/wavetrace/
```

**Comment by kesara** [comment]
setup.py file refers to README.rst but it's missing in tarball in https://pypi.python.org/pypi/wavetrace/3.0.0

**Comment by kesara** [comment]
Changes with merge request #20 and 3.0.1 release seems to fix the `README.rst` missing file issue, but 3.0.1 tarball is missing the `LICENSE` file.

```
Collecting wavetrace

  Downloading wavetrace-3.0.1.tar.gz (42kB)
    100% |████████████████████████████████| 51kB 5.4MB/s
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-_sao1guw/wavetrace/setup.py", line 7, in <module>
        with open('LICENSE') as f:
    FileNotFoundError: [Errno 2] No such file or directory: 'LICENSE'
```

**Comment by kesara** [comment]
Fixed by #21
